bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -472,7 +638,7 @@ spec:
                 - --leader-elect
                 - --ebpf-agent-image=$(RELATED_IMAGE_EBPF_AGENT)
                 - --flowlogs-pipeline-image=$(RELATED_IMAGE_FLOWLOGS_PIPELINE)
-                - --console-plugin-image=$(RELATED_IMAGE_CONSOLE_PLUGIN)
+                - --console-plugin-image=$(RELATED_IMAGE_CONSOLE_PLUGIN)"
```

Comment:
hmm? added double-quote is not intentional I guess?

Reply:
right

Reply:
fixed it removed the extra `"` at the end